### PR TITLE
en,zh: Optimize the doc about arm64 deployment

### LINKED ISSUE #1961: Optimize the doc about arm deployment.
## Change Request

**This repository is ONLY used to solve issues related to TiDB Operator documentation.
For other issues (related to TiDB, PD, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. Describe what you find is inappropriate or missing in the existing docs.

2. Describe your suggestion or addition.

3. Provide some reference materials (documents, websites, etc) if you could.

### PR DESCRIPTION
**Pull request by KanShiori**
### What is changed, added, or deleted? (Required)
Close #1961

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/deploy-cluster-on-arm64.md [hunk 1]
```diff
@@ -25,6 +29,10 @@ tidbBackupManagerImage: pingcap/tidb-backup-manager-arm64:v1.3.1
 
 ## 部署 TiDB 集群
 
+> **注意：**
```
Comment (DanielZhangQD) [inline zh/deploy-cluster-on-arm64.md:32]: Add two sections: 
>=5.4.2
<5.4.2
Similar for the TiDB Operator.

Comment (KanShiori) [reply zh/deploy-cluster-on-arm64.md]: Done

**Review by csuzhangxc — APPROVED** [review]

File: zh/deploy-cluster-on-arm64.md [hunk 2]
```diff
@@ -13,51 +13,55 @@ summary: 本文档介绍如何在 ARM64 机器上部署 TiDB 集群
...
-  namespace: ${cluster_namespace}
-spec:
-  version: "v5.4.1"
-  # ...
-  helper:
-    image: busybox:1.33.0
-  # ...
-  pd:
-    baseImage: pingcap/pd-arm64
+* 如果部署的 TiDB 集群大于或等于 v5.4.2 版本，那么跳过这一步。
```
Comment (ran-huang) [inline zh/deploy-cluster-on-arm64.md:30]: 同上。“跳过这一步”有歧义。

File: zh/deploy-cluster-on-arm64.md [hunk 3]
```diff
@@ -13,51 +13,55 @@ summary: 本文档介绍如何在 ARM64 机器上部署 TiDB 集群
 
 ## 部署 TiDB Operator
 
-在 ARM64 机器上部署 TiDB Operator 的步骤与[在 Kubernetes 上部署 TiDB Operator](deploy-tidb-operator.md) 的步骤相同。需要注意的是在 v1.3.1 及之前版本，在[自定义部署 TiDB Operator](deploy-tidb-operator.md#自定义部署-tidb-operator) 这一步，当获取到 `tidb-operator` chart 中的 `value.yaml` 文件后，你需要修改文件中的 `operatorImage` 与 `tidbBackupManagerImage` 字段为 ARM64 版本镜像。例如：
+* 如果部署的 TiDB Operator 大于或等于 v1.3.1 版本，那么跳过这一步。
```
[the remark this answers is not in the document]
Comment (KanShiori) [reply zh/deploy-cluster-on-arm64.md:16]: 是的

[the remark this answers is not in the document]
Comment (KanShiori) [reply zh/deploy-cluster-on-arm64.md]: Done

File: zh/deploy-cluster-on-arm64.md [hunk 4]
```diff
@@ -13,51 +13,55 @@ summary: 本文档介绍如何在 ARM64 机器上部署 TiDB 集群
...
 
-```yaml
-# ...
-operatorImage: pingcap/tidb-operator-arm64:v1.3.1
-# ...
-tidbBackupManagerImage: pingcap/tidb-backup-manager-arm64:v1.3.1
-# ...
-```
+* 如果部署的 TiDB Operator 小于 v1.3.1 版本，在 ARM64 机器上部署 TiDB Operator 的步骤与[在 Kubernetes 上部署 TiDB Operator](deploy-tidb-operator.md) 的步骤相同。唯一区别是，在[自定义部署 TiDB Operator](deploy-tidb-operator.md#自定义部署-tidb-operator) 这一步，当获取到 `tidb-operator` chart 中的 `value.yaml` 文件后，你需要修改文件中的 `operatorImage` 与 `tidbBackupManagerImage` 字段为 ARM64 版本镜像。
```
[the remark this answers is not in the document]
Comment (KanShiori) [reply zh/deploy-cluster-on-arm64.md]: Done

**Pull request merged**